cluster/aws.go

```diff
@@ -454,12 +534,9 @@ func (c *AWSCluster) UpdateCluster(request *components.UpdateClusterRequest) err
 		SecretId:         c.modelCluster.SecretId,
 		Status:           c.modelCluster.Status,
 		Amazon: model.AmazonClusterModel{
-			NodeSpotPrice:      c.modelCluster.Amazon.NodeSpotPrice,
-			NodeMinCount:       request.Amazon.MinCount,
-			NodeMaxCount:       request.Amazon.MaxCount,
-			NodeImage:          c.modelCluster.Amazon.NodeImage,
 			MasterInstanceType: c.modelCluster.Amazon.MasterInstanceType,
 			MasterImage:        c.modelCluster.Amazon.MasterImage,
+			NodePools:          updatedNodePools,
```

Comment:
This variable is just initialised but holds no value, is this right?:)

Reply:
Update part will be refactored in other patch